src/agent.py

```
@@ -259,59 +310,145 @@ async def fetch_long_term():
     return schemas.DialecticResponse(content=response[0]["text"])
 
 
-async def get_long_term_facts(
-    query: str, app_id: str, user_id: str, collection_id: str
-) -> list[str]:
+async def get_observations(
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)